main.py

```diff
@@ -26,8 +30,72 @@ def get_games_repo(request: Request) -> GameRepository:
     return request.state.game_repo
 
 
-# endpoing juguete, borralo cuando haya uno de verdad
 @app.get("/api/borrame")
 async def borrame(games_repo: GameRepository = Depends(get_games_repo)):
     games = games_repo.get_many(10)
     return {"games": games}
+
+
+class GameIn(BaseModel):  # Heredar de BaseModel
+    name: str
+    max_players: int
+    min_players: int
```

Comment:
Usa Pydantic para poner los rangos
```suggestion
    name: str = Field(min_length=1, max_length=64)
    max_players: int = Field(ge=2, le=4)
    min_players: int = Field(ge=2, le=4)
```